TourMate/TourMate/Frontend/Plan/Models/Activity.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 class Activity: Plan {
-    var location: Location?
+    var location: Location
```

Comment:
just curious why is the location compulsory now?

Reply:
To minimise the code from converting `Binding<Location?>` to `Binding<Location>` because `Location` is compulsory in `Trip`